src/Adapters/Push/OneSignalPushAdapter.js

```diff
@@ -134,7 +134,42 @@ export class OneSignalPushAdapter extends PushAdapter {
     if(data['uri']) {
       post['url'] = data['uri'];
     }
-
+    if(data['android_background_data'])
```

Comment:
it would be nicer with a reduce:

``` 
post = ['android_background_data', ...].reduce( (post, key) {
  if (data[key]) {
    post[key] = data[key];
    delete data[key];
  }
  return post;
}, post);
```